base/system/winlogon/lang/zh-TW.rc

```diff
@@ -14,24 +15,24 @@ END
 
 IDD_GINALOADFAILED DIALOGEX 58, 83, 231, 119
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "使用者介面失敗"
+CAPTION "使用者介面載入失敗"
 FONT 9, "新細明體"
 BEGIN
-    LTEXT "載入登入使用者介面的 DLL 檔案 %s 失敗。", IDC_GINALOADFAILED, 39, 16, 156, 24
-    LTEXT "請與系統管理員聯繫，以替換 DLL 檔案，或還原原 DLL。", IDC_GINALOADFAILEDCONTACT, 39, 53, 151, 25
+    LTEXT "載入登入使用者介面的 DLL 檔案 %s 載入失敗。", IDC_GINALOADFAILED, 39, 16, 156, 24
+    LTEXT "請與系統管理員聯繫，以替換 DLL 檔案，或還原初始的 DLL。", IDC_GINALOADFAILEDCONTACT, 39, 53, 151, 25
     DEFPUSHBUTTON "重新開機(&R)", 1, 80, 91, 68, 14
 END
 
 IDD_SYSSHUTDOWN DIALOGEX 50, 50, 180, 140
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_VISIBLE | WS_CAPTION | WS_POPUP
-CAPTION "System Shutdown"
-FONT 8, "MS Shell Dlg"
+CAPTION "系統關閉"
+FONT 8, "新細明體"
```

Comment:
possibly you should change the font size to 9.
see this:
https://reactos.org/wiki/RC_File_Standards

Reply:
ok, I changed the font size there to 9.